src/main/webapp/style.css

```diff
@@ -228,6 +228,11 @@
   width: 70%;
 }
 
+#add-maintainer-email-form {
```

Comment:
This change doesn't seem like it belongs in this PR. Please split these changes into different PRs for the future.

Reply:
I believe this change came from merge conflicts between this branch and master. Sorry!